lib/web_ui/lib/src/engine/browser_detection.dart

```diff
@@ -236,6 +236,11 @@ bool get isIOS15 {
       html.window.navigator.userAgent.contains('OS 15_');
 }
 
+/// A flag to check if the browser is iOS Safari.
```

Comment:
I know in other places we referred to getters like this as "flag" but typically a "flag" refers to a boolean environment variable or a compiler option. I'd just start with "Returns true if ..."